requirements/requirements-general.txt

```diff
@@ -1,11 +1,14 @@
-dacite==1.6.0
-fastjsonschema==2.15.3
-flask==2.0.3
+# update to newer version as soon as https://github.com/konradhalas/dacite/issues/218 is fixed
+dacite==1.7.0
+fastjsonschema==2.16.2
+# newer versions have problems with the concurrency tests
+flask==2.1.3
 gunicorn==20.1.0
-psycopg2==2.9.3
-redis==4.1.4
-opentelemetry-api
-opentelemetry-sdk
-opentelemetry-exporter-otlp
-opentelemetry-instrumentation-flask
-protobuf==3.20.1
+psycopg2==2.9.5
+redis==4.4.2
+
+# opentelemetry
+opentelemetry-api==1.15.0
+opentelemetry-sdk==1.15.0
+opentelemetry-exporter-otlp==1.15.0
+opentelemetry-instrumentation-flask==0.36b0
```

Comment:
Why do you remove protobuf at all? New versions in Pypi are on 4.x, dependabot proposed 3.20.2. It seem we don't use it anymore?

Reply:
protobuf is not used directly by us, but only as a nested dependency. We previously pinned it to a fixed version to circumvent an error introduced in 3.20.2 (see https://github.com/OpenSlides/openslides-datastore-service/commit/d303e31e68b24d944941eaa90904c342444c5c47), which has since been fixed, so we can remove the dependency again. See also https://github.com/OpenSlides/openslides-datastore-service/pull/216 for more info